libraries/WDT/src/WDT.cpp

```diff
@@ -0,0 +1,72 @@
+#include "WDT.h"
+
+// Constructor
+APM3_WDT::APM3_WDT()
+{
+    // The watchdog only runs off of the LFRC
+    am_hal_clkgen_control(AM_HAL_CLKGEN_CONTROL_LFRC_START, 0);
+    am_hal_wdt_init(&watchdogConfig);
+}
+
+// Configure the watchdog timer
```

Comment:
Add explanation line:
`//Options for clock per below, interrupt is number of ticks until ISR is called, reset is number of ticks until system reset`

Reply:
Added.